pkg/bootstrap/docker/up.go

```diff
@@ -714,14 +714,19 @@ func (c *ClientStartConfig) EnsureDefaultRedirectURIs(out io.Writer) error {
 
 // CheckAvailablePorts ensures that ports used by OpenShift are available on the Docker host
 func (c *CommonStartConfig) CheckAvailablePorts(out io.Writer) error {
+	userNamespace, err := c.DockerHelper().UserNamespaceEnabled()
```

Comment:
Why not ? (report about possible error)
If an error is returned by the docker info call, then cluster up should terminate. You can pass a dockerhelper to the NewRunHelper function.